lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -398,21 +398,21 @@ def success_from(response)
       def message_from(response)
         return response['status'] if success_from(response)
 
-        response['errorInformation']['message'] || response['message']
+        response.dig('errorInformation', 'message') || response['message']
```

Comment:
```suggestion
        response.dig('errorInformation', 'message') || response.dig('message')
```

Reply:
I rather keep it as response['message'] since it's not looking it will just return nil without breaking if message is not present